velox/type/Type.cpp

```diff
@@ -897,6 +899,16 @@ std::string IntervalDayTimeType::valueToString(int64_t value) const {
   return buf;
 }
 
+std::string IntervalYearMonthType::valueToString(int32_t value) const {
+  static const char* kIntervalFormat = "%d %d";
+  int32_t remainMonths = value;
```

Comment:
This could simply be
const int32_t years = value / kMonthInYear;
const int32_t remainMonths = value - years * kMonthInYear;